plot_app/plotting.py

```diff
@@ -753,56 +755,59 @@ def add_graph(self, field_names, legends, window='hann', window_length=256, nove
             delta_t = ((data_set['timestamp'][-1] - data_set['timestamp'][0]) * 1.0e-6) / len(data_set['timestamp'])
             sampling_frequency = int(1.0 / delta_t)
 
-            field_names_expanded = self._expand_field_names(field_names, data_set)
+            if sampling_frequency < 100:
+                self._had_error = True
+            else:
```

Comment:
Can you do:
```
if sampling_frequency < 100:
    self._had_error = True
    return
```
To avoid adding one more level of indentation for the following code?